protocol/communities/manager.go

```diff
@@ -623,6 +709,23 @@ func (m *Manager) ControlledCommunities() ([]*Community, error) {
 	return communities, nil
 }
 
+func (m *Manager) Owned() ([]*Community, error) {
+	communities, err := m.persistence.CommunitiesWithPrivateKey(&m.identity.PublicKey)
```

Comment:
This function name makes it sound like we it only returns communities for which we have the community's private key. Is that the case?
If so, maybe there is a problem, because you can be the owner of a community because of the owner token while not having its private key.

Maybe this is all just a little naming problem

Reply:
Yes, you would, but you would have the private key of the community, or rather, the contract will be set to your (or any public key you have access), and then you can `SetPrivateKey` of the community. 
So the function is checking that the community has private key set, and that you are the owner of that community (i.e the controlNode is the public key corresponding to the private key you have set on the community)
hope that it's clear, not sure if I explained it very clearly :)

Reply:
Ok I get it. The name is  a bit confusing when you know the old way of doing things, but it's ok